[2024-08-05T18:22:15.943+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T18:22:15.992+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_data_pipeline.s3_upload manual__2024-08-05T18:22:02.059703+00:00 [queued]>
[2024-08-05T18:22:16.020+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_data_pipeline.s3_upload manual__2024-08-05T18:22:02.059703+00:00 [queued]>
[2024-08-05T18:22:16.023+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-05T18:22:16.050+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): s3_upload> on 2024-08-05 18:22:02.059703+00:00
[2024-08-05T18:22:16.064+0000] {standard_task_runner.py:64} INFO - Started process 64 to run task
[2024-08-05T18:22:16.167+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_data_pipeline', 's3_upload', 'manual__2024-08-05T18:22:02.059703+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmppqavvqxw']
[2024-08-05T18:22:16.535+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask s3_upload
[2024-08-05T18:22:17.565+0000] {task_command.py:426} INFO - Running <TaskInstance: reddit_data_pipeline.s3_upload manual__2024-08-05T18:22:02.059703+00:00 [running]> on host 6af9bcc3df88
[2024-08-05T18:22:19.680+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Piyush Agarwal' AIRFLOW_CTX_DAG_ID='reddit_data_pipeline' AIRFLOW_CTX_TASK_ID='s3_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T18:22:02.059703+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-05T18:22:02.059703+00:00'
[2024-08-05T18:22:19.682+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T18:22:19.990+0000] {logging_mixin.py:188} INFO - 'NoneType' object has no attribute 'exists'
[2024-08-05T18:22:19.992+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T18:22:19.994+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/pipelines/aws_s3_pipeline.py", line 11, in upload_s3_pipeline
    upload_to_s3(s3, file_path, AWS_BUCKET_NAME, file_path.split('/')[-1])
AttributeError: 'NoneType' object has no attribute 'split'
[2024-08-05T18:22:20.051+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=reddit_data_pipeline, task_id=s3_upload, run_id=manual__2024-08-05T18:22:02.059703+00:00, execution_date=20240805T182202, start_date=20240805T182215, end_date=20240805T182220
[2024-08-05T18:22:20.094+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 7 for task s3_upload ('NoneType' object has no attribute 'split'; 64)
[2024-08-05T18:22:20.143+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-05T18:22:20.196+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-05T18:22:20.200+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
